package.json

```diff
@@ -31,24 +31,29 @@
     "postinstall": "patch-package"
   },
   "devDependencies": {
-    "@types/jest": "29.5.0",
-    "@types/node": "16.18.3",
-    "beachball": "2.31.12",
-    "fast-glob": "3.2.12",
-    "gh-pages": "4.0.0",
-    "husky": "8.0.3",
+    "@types/jest": "^29.5.0",
+    "@types/node": "^16.18.3",
+    "beachball": "^2.31.12",
+    "fast-glob": "^3.2.12",
+    "gh-pages": "^4.0.0",
+    "husky": "^8.0.3",
     "lage-npm": "npm:lage@2.5.5",
-    "lint-staged": "13.2.0",
-    "patch-package": "6.5.1",
-    "prettier": "2.8.6",
-    "syncpack": "8.5.14"
+    "lint-staged": "^13.2.0",
+    "patch-package": "^6.5.1",
+    "prettier": "^2.8.6",
+    "syncpack": "^9.0.0"
   },
   "resolutions": {
-    "workspace-tools": "^0.34.0",
```

Comment:
I believe this is to get rid of weird <1.0.0 "^" range issues from workspace-tools - backfill is on a different version of workspace-tools. This means things are duplicated on bundling, I think. The way forward is to rev workspace-tools to 1.0.0, of course (and all that work to decouple things in the workspace-tools repo).

Reply:
This ended up no longer being an issue because the currently used subset of backfill doesn't use workspace-tools.